CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+Ensure you have read the [FAQ](http://django-autocomplete-light.readthedocs.org/en/stable-2.x.x/faq.html).
+
+Before opening an issue, ensure you have:
+
+- clear steps to reproduce the bug,
+- a fork of the repo with an additionnal app in test_project to reproduce your
+  issue out-of-the-box,
+
+Do **not** bother:
+
+- pasting partial code in the issue text, ie. only forms and admin without
+  models, etc, etc... Actually don't bother posting code at all: fork the repo,
+  add example app, make a PR. We're not supporting anything that has not an
```

Comment:
"add an example app"

"We are more likely to fix/support something with an example app."